bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioManagerImpl.java

```diff
@@ -90,51 +91,80 @@ public void play(AudioStream audioStream) {
 
     @Override
     public void play(AudioStream audioStream, String sinkId) {
+        play(audioStream, sinkId, null);
+    }
+
+    public void play(AudioStream audioStream, String sinkId, PercentType volume) {
         if (audioStream != null) {
             AudioSink sink = getSink(sinkId);
 
             if (sink != null) {
-                try {
-                    sink.process(audioStream);
-                } catch (UnsupportedAudioFormatException | UnsupportedAudioStreamException e) {
-                    logger.error("Error playing '{}': {}", audioStream.toString(), e.getMessage());
+                // get current volume
+                PercentType oldVolume = getVolume(sinkId);
+                // set notification sound volume
+                if (volume != null) {
+                    setVolume(volume, sinkId);
+                }
+//                if (sink.getSupportedStreams().contains(audioStream.getClass())) {
```

Comment:
If not required, you can actually remove such unused code comment blocks

Reply:
I don't want to remove them, I want to uncomment them. If I do Travis fails (see https://travis-ci.org/eclipse/smarthome/builds/331787920). But I'm not sure why. May I ask you to help me with that? I somehow have the feeling that it is related to your previous commt on the `audioSink` is null issue
(https://github.com/eclipse/smarthome/pull/4963/files#r163219285).

Reply:
You mean when you remove them travis fails?

Reply:
Yes, I exaclty. But I found the problem now. The condition `sink.getSupportedStreams().contains(audioStream.getClass())` is to sloppy.